tests/should_succeed/file.jou

```diff
@@ -5,7 +5,7 @@ def write_hello_123() -> None:
     f = fopen("tmp/tests/hello.txt", "w")
     if f == NULL:
         printf("can't write tmp/tests/hello.txt\n")
-        abort()
+        exit(1) # abort() would make a core dump when comparing compilers
```

Comment:
```suggestion
        exit(1)  # on NetBSD, abort() would make a core dump when comparing compilers
```

Reply:
Not specific to NetBSD but to your `ulimit` setting.